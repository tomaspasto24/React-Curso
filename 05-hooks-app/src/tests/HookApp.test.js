import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import { HookApp } from '../HookApp';

describe('Pruebas sobre HookApp.', () => {

    test('Debe mostrar el componente correctamente.', () => {

        const wrapper = shallow( <HookApp /> );
        
        expect(wrapper).toMatchSnapshot();
    });

});