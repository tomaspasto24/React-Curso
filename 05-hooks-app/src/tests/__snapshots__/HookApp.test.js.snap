// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre HookApp. Debe mostrar el componente correctamente. 1`] = `
<Fragment>
  <h1>
    Hola Mundo
  </h1>
</Fragment>
`;
